skbeam/core/mask.py

```diff
@@ -79,7 +79,7 @@ def bad_to_nan_gen(images, bad):
             yield im
 
 
-def threshold_mask(images, threshold, mask=None):
```

Comment:
Reading what this does, it probably should have the phrase 'gen' or 'filter' or something to indicate that it is streaming / lazy.